src/prototype_creep_routing.js

```diff
@@ -23,9 +23,11 @@ Creep.prototype.getRoute = function() {
 
       if (Memory.rooms[roomName] && Memory.rooms[roomName].state == 'Occupied') {
         //         console.log(`Creep.prototype.getRoute: Do not route through occupied rooms ${roomName}`);
-        if (config.allowRoutingThroughFriendRooms && friends.indexOf(Memory.rooms[roomName].player) > -1) {
+        if (config.path.allowRoutingThroughFriendRooms && friends.indexOf(Memory.rooms[roomName].player) > -1) {
+          console.log('routing through friendly room' + roomName);
           return 1;
         }
+        console.log('Not routing through enemy room' + roomName);
```

Comment:
I would remove the log output. For debugging the other should be enough. This one increases the noise.